rma/models/rma.py

```diff
@@ -670,21 +661,94 @@ def action_rma_send(self):
         }
 
     def _add_message_subscribe_partner(self):
+        self.ensure_one()
         if self.partner_id and self.partner_id not in self.message_partner_ids:
             self.message_subscribe([self.partner_id.id])
 
+    def _product_is_storable(self, product=None):
+        product = product or self.product_id
+        return product.type in ["product", "consu"]
+
+    def _prepare_procurement_group_values(self):
+        return {
+            "move_type": "direct",
+            "partner_id": self and self.partner_shipping_id.id or False,
+            "name": self and ", ".join(self.mapped("name")) or False,
+        }
+
+    def _prepare_procurement_values(
```

Comment:
For better understanding and use the usual suffix as well (applicable this second part to all the code):

```suggestion
    def _prepare_common_procurement_vals(
```